wiki/People/fr.md

```diff
@@ -1,8 +1,4 @@
----
-outdated: true
----
-
 # L'Équipe
```

Comment:
this is supposed to be the title of "The Team", right?

Reply:
Yes